core/src/test/java/com/gentics/mesh/core/schema/field/DateFieldMigrationTest.java

```diff
@@ -35,15 +33,6 @@ public void testRemove() throws Exception {
 		removeField(CREATEDATE, FILL, FETCH);
 	}
 
-	@Override
-	@Test
-	public void testRename() throws Exception {
```

Comment:
Check why this was removed.

Reply:
A custom script was used for renaming fields.